### DISCUSSION TITLE
Can I join GitHub Education without sharing location?

### DISCUSSION THREAD

**Discussion by EsMariaW** [community/community#136901] [GitHub Education]
### Hi everyone,

Hi everyone,
I am a university student interested in using GitHub Copilot. I understand I need to join GitHub Education for access, but the sign-up process requires enabling location services, which I don’t use. Is there a way to join GitHub Education without sharing my location?
Thanks so much for your help!

**Comment by ThisIs-Developer** [comment]
Hi,
1. **Contact GitHub Support** for alternative verification methods: [GitHub Support](https://support.github.com/contact).
2. **Provide documentation** like a student ID or university letter.
3. **Check privacy settings** or try a different device with location services enabled.

If this helps, please mark as answer!😴

**Reply by NeutralKaon** [reply]
I don't want to share my location on privacy grounds – Microsoft knows too much anyway (and we pay you many millions a year) -- and will be using support. I'm an associate professor and do _not_ want to normalise the provision of GPS tracking data in transactions like these. Thanks for keeping the alternative alive at least...

**Reply by NeutralKaon** [reply]
Just to say, this doesn't work. Selecting the option from the dropdown box of "you have rejected me" leads to this message: 

>  Individual manual reviews and status updates are no longer provided.
> It appears that you are looking to contact us regarding your GitHub Global Campus application. We no longer provide individual manual reviews or application status updates. 

Great. Fantastic customer support from one of the richest companies in the world.

**Reply by rojikada** [reply]
What a breach of privacy. It is not worth my data, thank you GitHub!

**Comment by davevad93** [comment]
Hi @EsMariaW ,

I don't think there's any way to apply without enabling your location because they use it to verify that you're located in the same area as your school.

This is what's stated in the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352#not-near-any-campus-location-for-selected-school):

- ***In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program.***

For more info, take a look here:

- [You appear to not be near any campus location for the school you have selected.](https://github.com/orgs/community/discussions/111352#not-near-any-campus-location-for-selected-school)

**Comment by EsMariaW** [comment]
Hi everyone,
I just wanted to say thank you so much for taking the time to answer my question! I really appreciate it. Hope you have a great day!

**Reply by davevad93** [reply]
You're welcome @EsMariaW , have a great day too! 👍

**Comment by Welding-Torch** [comment]
Facing the same issue.

**Comment by krillmango** [comment]
I disabled location services and other malware with a fresh install of [WindowsXlite](https://windowsxlite.com/); it uses my very valid windows license

Consider a mobile platform with location spoofing ability.  NINTENDO played this [war of attrition](https://duckduckgo.com/?q=gnss+spoofing+hardware&ia=outsmartgithubtoo) with us and lost.

**Comment by RahulRana9520** [comment]
GitHub Education may use location as part of student verification, so there usually isn’t a way to skip it if they specifically require it during sign-up. The better option is to contact GitHub Education Support and ask whether they can verify your student status using documents like a student ID or enrollment proof instead of location access
